slam_toolbox/lib/karto_sdk/src/Mapper.cpp

```diff
@@ -1605,7 +1605,11 @@ namespace karto
     // only attach link information if the edge is new
```

Comment:
First off, just linting things, place space between the `=` and don't use `auto` since the rest of this file doesn't make use of it for styling consistency.

Reply:
I've pushed a new commit.